updated provisioners reference topics (#13273)
hashicorp/packer: website/content/docs/provisioners/shell.mdx

---
description: |

>   The `shell` Packer provisioner provisions machines built by Packer using shell
>   scripts. Use the `shell` provisioner install and configure software on a machine.

Reviewer: ```suggestion
  The `shell` Packer provisioner runs shell scripts on the machine Packer builds. Use the `shell` provisioner to install and configure software on a machine.
```

What do you think about this tweak to the wording? "Provisions machines using shell scripts" feels a little ambiguous to me, but maybe that's just me :)

page_title: shell provisioner reference
---

<BadgesHeader>
  <PluginBadge type="official" />
</BadgesHeader>

# `shell` provisioner

> The `shell` Packer provisioner provisions machines built by Packer using shell
> scripts. Use the `shell` provisioner install and configure software on a machine.

Reviewer: ```suggestion
The `shell` Packer provisioner runs shell scripts on the machine Packer builds. Use the `shell` provisioner to install and configure software on a machine.
```

Mirroring the suggestion above.
